opentech/apply/funds/management/commands/migrate_rr_applications.py

```diff
@@ -244,8 +245,14 @@ def handle(self, *args, **options):
                 self.process(id)
 
     def process(self, id):
+        User = get_user_model()
         node = self.data[id]
 
+        try:
+            lead = User.objects.get(full_name="Lindsay Beck")
```

Comment:
This migration depends on https://github.com/OpenTechFund/opentech.fund/blob/fa10f297856d4358eeeda87033a848a293447522/opentech/apply/users/management/commands/migrate_users.py

But a good idea to wrap it in try/except. 

Note that the submission will inherit the lead from the Round - https://github.com/OpenTechFund/opentech.fund/blob/fa10f297856d4358eeeda87033a848a293447522/opentech/apply/funds/management/commands/migrate_rr_applications.py#L221